gtests/net/mptcp/sockopts/sockopt_cork_nodelay.pkt

```diff
@@ -0,0 +1,50 @@
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
+
++0.0  socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0.0  setsockopt(3, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
++0.0  setsockopt(3, SOL_TCP, TCP_CORK, [1], 4) = 0
+
++0    bind(3, ..., ...) = 0
++0    listen(3, 1) = 0
```

Comment:
out of curiosity, why did you take the view of the server?
It is fine like that but usually, we start with the client side, it's usually "easier" to understand :)

Reply:
There are three main reasons for this:

1) mptcp/sockopts/sockopt_cork_nodelay.pkt is mostly derived from tcp/nagle/sockopt_cork_nodelay.pkt -- that has a server perspective.
2) I wanted to ensure that cloned (accepted) sockets inherit cork/nodelay status from parent, as Paolo mentioned in his email.
3) I mostly do server work, and that perspective is a bit more familiar :)